cfme/provisioning.py

```diff
@@ -217,6 +217,9 @@ class schedule(Tab):  # noqa
             ('automatic_placement', ui.Input('environment__placement_auto')),
 
             # Cloud
+            ('cloud_tenant', {
+                version.LOWEST: ui.Select('select#environment__cloud_tenant'),
+                '5.5': AngularSelect('environment__cloud_tenant')}),
```

Comment:
**Required**
We don't need to account for anything lower than 5.5, this can just be the AngularSelect.